### DISCUSSION TITLE
How I can run gatus without containers?

### DISCUSSION THREAD

**Discussion by waleedsaud** [TwiN/gatus#750] [Q&A] [answered]
I tried to run the gatus binary but I couldn't find the comand line options and there is no help option . Also I couldn't find any docs related to it.

**Comment by amai2012** [comment]
Well, the container itself is documented via the [Dockerfile](https://github.com/TwiN/gatus/blob/master/Dockerfile) ...

**Comment by TwiN** [comment, marked as answer]
Currently, binaries are not published alongside Gatus releases as Gatus is intended to be run as a Docker container, but if you _really_ want to run it as a binary, you'll have to build the binary yourself, which means you'll need to install Go, clone this repository and run the command provided in https://github.com/TwiN/gatus#installing-as-binary

**Comment by waleedsaud** [comment]
Thank you , I used go to build it as a binary and it worked .
